### PR TITLE
Added  Recursion Basic Questions

### PR DESCRIPTION
**Pull request by KaifKhan7393**

### CODE REVIEW

File: recursion/Basic Questions/Factorial.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+def factorial(n):
+    if n == 0:
+        return 1
```
Comment (fms12) [inline recursion/Basic Questions/Factorial.py:3]: please add comments inside all files for better understanding

**Review by fms12 — CHANGES_REQUESTED** [review]

File: recursion/Basic Questions/Factorial.py (hunk 1 above)
Comment (KaifKhan7393) [reply recursion/Basic Questions/Factorial.py:3]: ok

**Comment by KaifKhan7393** [comment]
I have added the comments on all these files

**Review by fms12 — APPROVED** [review]

**Comment by fms12** [comment]
marked complete in the issue

**Pull request merged**